OpenDreamClient/Rendering/DreamViewOverlay.cs

```diff
@@ -208,18 +207,23 @@ private void ProcessIconComponents(DreamIcon icon, Vector2 position, EntityUid u
             icon.Appearance.Transform[4], icon.Appearance.Transform[5]
         );
 
-        if (parentIcon != null) {
+        if (parentIcon != null)
+        {
```

Comment:
The codebase style is to keep braces on the same line.
```suggestion
        if (parentIcon != null) {
```